tests/test_utils.py

```diff
@@ -1255,7 +1255,11 @@ def test_detect_travis():
 
     Simply check it passes without exception.
     """
-    _ = utils.env_detection.running_on_travis()
+    on_travis = utils.env_detection.running_on_travis()
+    if "ON_TRAVIS" in os.environ:
+        assert on_travis
+    else:
+        assert not on_travis
```

Comment:
Not sure about this. I think it was better before simply checking that it returns. Maybe could add a check that it is a boolean value. This looks like it will easily break as a test and isn't actually testing anything,

Reply:
`utils.env_detection.running_on_travis()` is dependent on Travis variables. So I have forced a test which is relying on a variable which is manually added in case the other variables fail.
This will allow the test to pass locally as well otherwise tests will fail locally.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/292/commits/22768389760785e09e96e83bb7d1a33e6fa2c2e7